x/evmutil/types/conversion_pair.go

```diff
@@ -0,0 +1,89 @@
+package types
+
+import (
+	bytes "bytes"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// NewConversionPair returns a new ConversionPair.
+func NewConversionPair(address InternalEVMAddress, denom string) ConversionPair {
+	return ConversionPair{
+		KavaERC20Address: address.Address.Bytes(),
+		Denom:            denom,
+	}
+}
+
+// GetAddress returns the InternalEVMAddress of the Kava ERC20 address.
+func (pair ConversionPair) GetAddress() InternalEVMAddress {
+	return NewInternalEVMAddress(common.BytesToAddress(pair.KavaERC20Address))
+}
+
+// Validate returns an error if the ConversionPair is invalid.
+func (pair ConversionPair) Validate() error {
+	if pair.Denom == "" {
+		return errors.New("denom cannot be empty")
+	}
```

Comment:
nit: `sdk.ValidateDenom` is used to validate denoms in other modules. I'd probably use it here in as we intend to send these coins to other modules.